Libs/Core/ctkUtils.h

```diff
@@ -196,6 +196,24 @@ QString CTK_CORE_EXPORT absolutePathFromInternal(const QString& internalPath, co
 /// Absolute path can be retrieved from an internal path using absolutePathFromInternal function.
 QString CTK_CORE_EXPORT internalPathFromAbsolute(const QString& absolutePath, const QString& basePath);
 
+
+/// \ingroup Core
+/// \brief Calls QTextStream::flush() on \a stream and returns \a stream.
+///
+/// This method was added so that the same code compiles without deprecation warnings
+/// pre and post Qt 5.14.
+///
+/// For Qt >= 5.14, it is equivalent to using \a Qt::flush;
+CTK_CORE_EXPORT QTextStream &flush(QTextStream &stream);
```

Comment:
Adding the export first is required to avoid this warning and link error on linux using gcc :

```
In file included from /home/jcfr/Projects/CTK/Libs/Core/ctkErrorLogFDMessageHandler.cpp:29:0:
/home/jcfr/Projects/CTK/Libs/Core/ctkUtils.h:207:56: warning: ‘visibility’ attribute ignored on non-class types [-Wattributes]
 QTextStream & CTK_CORE_EXPORT flush(QTextStream &stream);

[...]

/home/jcfr/Projects/CTK/Libs/Core/Testing/Cpp/ctkUtilsCopyDirRecursivelyTest1.cpp:47: undefined reference to `ctk::endl(QTextStream&)'
CMakeFiles/CTKCoreCppTests.dir/ctkUtilsTest4.cpp.o: In function `(anonymous namespace)::createFile(int, QDir const&, QString const&, QString const&)':
/home/jcfr/Projects/CTK/Libs/Core/Testing/Cpp/ctkUtilsTest4.cpp:48: undefined reference to `ctk::endl(QTextStream&)'                                                        ^
```

Reply:
For reference, similar symbols are exported with the EXPORT in the front in Qt itselt:

* Qt 5.10: https://github.com/qt/qtbase/blob/5.10/src/corelib/io/qtextstream.h#L235-L245

* Qt 5.15: https://github.com/qt/qtbase/blob/9db7cc79a26ced4997277b5c206ca15949133240/src/corelib/serialization/qtextstream.h#L231-L239

Reply:
https://github.com/commontk/CTK/pull/1009 updates the export convention across CTK